### PR TITLE
Add maintainers to fdb, eckit, ecbuild, metkit, eccodes

### PR DESCRIPTION
**Pull request by victoria-cherkas**
This PR adds maintainers @victoria-cherkas and @dominichofer to the following packages: 
- fdb
- eckit
- ecbuild
- metkit
- eccodes

We depend heavily on these packages and their latest releases, and we would add/review new releases and updates when required.

### CODE REVIEW

**Comment by dominichofer** [comment]
Yes, LGTM.

File: var/spack/repos/builtin/packages/eccodes/package.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class Eccodes(CMakePackage):
     git = "https://github.com/ecmwf/eccodes.git"
     list_url = "https://confluence.ecmwf.int/display/ECC/Releases"
 
-    maintainers("skosukhin")
+    maintainers("skosukhin", "victoria-cherkas", "dominichofer")
```
Comment (climbfuji) [inline var/spack/repos/builtin/packages/eccodes/package.py]: Can you add me here as well, please?

**Review by tldahlgren — DISMISSED** [review]
LGTM.

**Comment by tldahlgren** [comment]
Reminder: https://spack.readthedocs.io/en/latest/packaging_guide.html#maintainers

File: var/spack/repos/builtin/packages/eccodes/package.py (hunk 1 above)
Comment (victoria-cherkas) [reply var/spack/repos/builtin/packages/eccodes/package.py]: Sure, done: https://github.com/spack/spack/pull/41433/commits/a0bae1381a6d82c4ad931b98f3c336c465f32382

**Review by tldahlgren — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]
Thanks!

**Comment by skosukhin** [comment]
I only would like to ask you not to abuse this to quickly bypass my reviews with changes to `eccodes` (I don't care that much about the rest of the packages).

**Comment by tldahlgren** [comment]
> I only would like to ask you not to abuse this to quickly bypass my reviews with changes to `eccodes` (I don't care that much about the rest of the packages).

Hmmm.  I will *try* to remember when I review.

**Comment by tldahlgren** [comment]
> I only would like to ask you not to abuse this to quickly bypass my reviews with changes to `eccodes` (I don't care that much about the rest of the packages).

Do you want to approve this PR at this point?

**Pull request merged**